website/content/docs/connect/proxies/envoy.mdx

```diff
@@ -122,7 +123,6 @@ until the LTS release reaches its end of maintenance.
 
 | Consul Version | Default `consul-dataplane` Version   | Other compatible `consul-dataplane` Versions |
 | -------------- | -------------------------------------|----------------------------------------------|
-| 1.19.x Ent     | 1.5.x (Envoy 1.29.x)                 | 1.4.x (Envoy 1.28.x)                         |
```

Comment:
I'm not sure why 1.19 was included in this table of LTS releases, but I removed it

Reply:
🍝